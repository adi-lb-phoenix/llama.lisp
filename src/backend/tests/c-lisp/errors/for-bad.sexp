(c-lisp
    (define ((main void))
        (declare i int)
        (set i 0)
        (for ((lt i 10) (set (add i 1))
            (call print i)))))
