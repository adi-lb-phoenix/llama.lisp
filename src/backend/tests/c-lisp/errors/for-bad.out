Error in statement: ['for', [['lt', 'i', 10], ['set', ['add', 'i', 1]], ['call', 'print', 'i']]]
CodegenError: Bad for statement: ['for', [['lt', 'i', 10], ['set', ['add', 'i', 1]], ['call', 'print', 'i']]]
